cmake/os/AIX.cmake

```diff
@@ -14,20 +14,24 @@
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA 
 
 
+# This will never work with AIX, so switching off per default
+option(WITH_UNIT_TESTS "WITH_UNIT_TESTS" OFF)
```

Comment:
Ok, it's nice to get them fixed, but I can live with having them disabled for now. Just put less restrictive comment, like "to be fixed later".